### PR TITLE
Pull request for #67: Create release/lg for large files

### PR DESCRIPTION
**Pull request by stanfordquan**
Please see #67 for details! Thanks Chuan.

 * Large files should be put into `/release/lg/`, where they will not be versioned. Small files should be put in `/release/` or subfolders and versioned with git(-lfs).

### CODE REVIEW

**Comment by yuchuan2016** [comment]
@stanfordquan , I will review this tomorrow. Before that, can you delete the `sconscript.log` and `state_of_repo.log` at the root of the repo? I think they should live in two sub-directories.

File: README.md [hunk 1]
```diff
@@ -98,7 +98,11 @@ We don't have a custom builder for LaTeX. You can still write in it, but you wil
...
 
-Every file intended for release should be added to the `release` directory. Files not intended for release to GitHub should be added to `.gitignore`. Our tool will transfer everything in `release` to the local destination and create a [GitHub release](https://help.github.com/articles/creating-releases/) with all the versioned files—those not added to `.gitignore`—in `release`. 
+Every file intended for release should be added to the `release` directory. Files not intended for release to GitHub should be added to `.gitignore`. Our tool will transfer everything in `release` to the local destination and create a [GitHub release](https://help.github.com/articles/creating-releases/) with all the versioned files—those not added to `.gitignore`—in `release`.
+
+##### What if there are large files I need to put in `release` but cannot be versioned?
+
+For large files to be released, our protocol is to keep them in a designated subfolder named `release/lg`. By default, `release/lg` is in `.gitignore`. `release/lg` will be included in the local destination release.
```
Comment (yuchuan2016) [inline README.md]: Can we add something like "when you make a release following our custom tool" in the end, just to emphasize these large files won't get versioned or released to local unless users manually release them.

**Comment by yuchuan2016** [comment]
Another thing is that, should we also update the tables/figures in `paper_slides/input`, since they have been modified in `analysis/release`?

**Comment by stanfordquan** [comment]
Done all 3 things, thanks @yuchuan2016 !

**Comment by yuchuan2016** [comment]
@stanfordquan , shall we run `scons` in `paper_slides` to rebuild paper and slides, since the input has been modified?

**Pull request merged**